clusterloader2/docs/test_suite_design.md

```diff
@@ -0,0 +1,50 @@
+# Cluster loader - TestSuite
+
+## Background
+
+Since 2019 cluster loader become main tool for kubernetes e2e performance tests.
+It is a very convenient tool for running single test case, but not for running
+test bundles. Currently it is impossible to run the same test
+with two different parameter sets in a single job run.
+This forces developers to create separate job configs for almost each test.
+
+## TestSuite
+
+TestSuite is a list of test scenarios running in a single test job.
+Tests will be executed by cluster loader in the order as provided.
```

Comment:
Do we really need this section? This explanation doesn't bring any value.
I would change to sth like

# Test suite definition
We define test suite as following:

```
type TestSuite []TestScenario

type TestScenario struct {
  ...
```

Those two lines doesn't provide any value. In general - the shorter the doc is the better if it contains all the information needed.

Reply:
Done.